package.json

```diff
@@ -18,43 +24,35 @@
     "react component"
   ],
   "scripts": {
+    "build": "BABEL_ENV=production rollup -c",
     "codecov": "codecov",
     "commitmsg": "commitlint -e $GIT_PARAMS",
-    "build-css": "node-sass src/CookiePolicyBanner/_cookie-policy-banner.scss -o build --importer node_modules/node-sass-import",
-    "build-styles": "npm run build-css && npm run copy-scss",
-    "compile-prod": "npm run transpile-prod && npm run build-styles",
-    "compile-dev": "npm run transpile-dev && npm run build-styles",
-    "copy-scss": "copyfiles --flat src/CookiePolicyBanner/_cookie-policy-banner.scss build/",
     "deploy-storybook-ci": "storybook-to-ghpages --ci",
-    "is-es5": "es-check es5 ./build/*.js",
+    "is-es5": "es-check es5 ./build/index.js",
     "lint": "eslint --ext .jsx,.js .",
-    "test": "jest --coverage --no-cache",
-    "prepublishOnly": "npm run compile-prod",
-    "css": "node-sass src/CookiePolicyBanner/_cookie-policy-banner.scss -o build --importer node_modules/node-sass-import",
+    "test": "BABEL_ENV=test jest --coverage --no-cache",
+    "prepare": "npm run build",
     "semantic-commit": "commit",
     "semantic-release": "semantic-release",
     "start": "start-storybook -p 3003",
-    "transpile-dev": "BABEL_ENV=development && npm run transpile",
-    "transpile-prod": "BABEL_ENV=production && npm run transpile",
-    "transpile": "rm -rf build/ && babel src/ -d build/ --copy-files --ignore node_modules,'**/*.stories.jsx','**/*.stories.js','**/*.scss'",
     "travis-deploy-once": "travis-deploy-once"
   },
-  "dependencies": {
-    "@edx/edx-bootstrap": "^1.0.0",
-    "@edx/paragon": "^3.0.5",
-    "classnames": "^2.2.5",
-    "prop-types": "^15.6.1",
-    "universal-cookie": "^2.1.2"
-  },
+  "dependencies": {},
```

Comment:
I moved all `dependencies` to `peerDependencies` and `devDependencies`. This will drastically reduce this library's bundle size.